### PR TITLE
Removed createDirectory() call from command-create.js

### LINKED ISSUE #67: Creating an alias throws mkdir fault
Installing squash by following the given steps in the tutorial, 
when I try to add an alias for a command it throws an error, linking the screenshot below.

screenshot 2019-02-14 at 2 33 15 pm

it tries to create a `local/.squash` file but finds it already there.
Is this error specific to my system or something which could be fixed.

OS: MacOS 10.14.3

### PR DESCRIPTION
**Pull request by co16353sidak**
Fixes #67 

Check everything which applies

- [x] I have added the issue number for which this pull request is created.

### CODE REVIEW

**Comment by arshadkazmi42** [comment]
Some tests are failing. Can you check it.
You can run `yarn test` in your local machine to test the tavis test cases.
And `mocha tests` to test all the tests cases
Both these tests should pass.

File: lib/file-operations/command-create.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ const commandCreate = (alias, command) => {
   try {
 
     // Creating squash config directory
-    directoryCreate();
+    // directoryCreate(); commented out as it is duplicate call
```
Comment (arshadkazmi42) [inline lib/file-operations/command-create.js]: Do not comment it out.
You can remove this code directory.
We do not want to keep the unused code in the codebase.
Also, if we are not using directoryCreate() function you have to remove the import also, so the lints will not fail

Comment (co16353sidak) [reply lib/file-operations/command-create.js]: Removed the call as advised.

**Comment by co16353sidak** [comment]
> Some tests are failing. Can you check it.
> You can run `yarn test` in your local machine to test the tavis test cases.
> And `mocha tests` to test all the tests cases
> Both these tests should pass.

**Comment by arshadkazmi42** [comment]
Build is failing. Can you try running tests in your local system.
You can check logs here
https://travis-ci.com/arshadkazmi42/squash/builds/100972190

**Comment by co16353sidak** [comment]
@arshadkazmi42 
`error  'directoryCreate' is assigned a value but never used  no-unused-vars`
Getting this fault.

From what I can concur, this could be due to

**We are somewhere in the code, creating an instance of `directoryCreate()`, removing the call is causing this error to throw.**

**Comment by co16353sidak** [comment]
Removing the line 
`const directoryCreate = require('./directory-create');`
https://github.com/arshadkazmi42/squash/blob/6a318c5c833161eb9615ca93e560e335af0a390c/lib/file-operations/command-create.js#L2

Makes `yarn test` pass

But `mocha tests` still fails.

File: lib/file-operations/command-create.js [hunk 2]
```diff
@@ -1,16 +1,12 @@
 const fs = require('fs');
-const directoryCreate = require('./directory-create');
+// const directoryCreate = require('./directory-create');
```
[the remark this answers is not in the document]
Comment (co16353sidak) [reply lib/file-operations/command-create.js]: Had commented out to check linting tests

Deleted them now.

[the remark this answers is not in the document]
Comment (arshadkazmi42) [reply lib/file-operations/command-create.js]: You can check it in your local.
`yarn lint` // checks for lints
`yarn test` // runs test.js tests

[the remark this answers is not in the document]
Comment (co16353sidak) [reply lib/file-operations/command-create.js]: `yarn lint` and `yarn test` both run successfully on local.

screenshot 2019-02-15 at 12 38 21 pm

screenshot 2019-02-15 at 12 38 40 pm

**Comment by arshadkazmi42** [comment]
> Removing the line
> `const directoryCreate = require('./directory-create');`
> 
> [squash/lib/file-operations/command-create.js](https://github.com/arshadkazmi42/squash/blob/6a318c5c833161eb9615ca93e560e335af0a390c/lib/file-operations/command-create.js#L2)
> 
> Line 2 in [6a318c5](/arshadkazmi42/squash/commit/6a318c5c833161eb9615ca93e560e335af0a390c)
>  const directoryCreate = require('./directory-create'); 
> 
> Makes `yarn test` pass
> 
> But `mocha tests` still fails.

The error was not related to your fix. It was related to some other test.
I have fixed it for now and updated your PR.
I will get this merged once TravisCI build is passed

**Comment by co16353sidak** [comment]
Ohh, that is relieving because I was completely unaware as to how mocha is failing. 😌

**Comment by co16353sidak** [comment]
@arshadkazmi42 Shall i add my name and photo in the contributors section in README? contributing here was a learning experience and would love some representation.

**Comment by arshadkazmi42** [comment]
Yes.
I use this CLI tool to generate contributors list.
You can install this tool and generate the Hall of Fame and replace the existing hall of fame.
The cli tool will handle all formatting and linting
https://github.com/arshadkazmi42/rhof-cli

**Pull request merged**